scprep/run/dyngen.py

```diff
@@ -0,0 +1,241 @@
+    
+    DyngenSimulate is a Python wrapper for the R package Dyngen. Default values obtained from Github vignettes. For more details, read about Dyngen on Github_.
+
+    .. _Github: https://github.com/dynverse/dyngen
+
+    Parameters
+    ----------
+    backbone: string
+           Backbone name from dyngen list of backbones.
+           Get list with get_backbones()).
+    num_cells: int, optional (default: 500)
+           Number of cells.
+    num_tfs: int, optional (default: 100)
+           Number of transcription factors. The TFs are the main drivers of the molecular changes in the simulation. A TF can only be regulated by other TFs or itself.
+           
+           NOTE: If num_tfs input is less than nrow(backbone$module_info), Dyngen will default to nrow(backbone$module_info). This quantity varies between backbones and with each run (without seed). However, in general, it is less than 75. It is recommended to input num_tfs >= 100 to stabilize the output.
+           
+    num_targets: int, optional (default: 50)
+           Number of target genes. Target genes are regulated by a TF or another target gene, but are always downstream of at least one TF. 
+    num_hks: int, optional (default: 25)
+           Number of housekeeping genees. Housekeeping genes are completely separate from any TFs or target genes.
+    simulation_census_interval: int, optional (default: 10)
+           Allows storing the abundance levels of molecules only after a specific interval has passed since the previous census. By setting the census interval to 0, the whole simulation’s trajectory is retained but many of these time points will contain very similar information.
+    compute_cellwise_grn: boolean, optional (default: False)
+           Whether or not to compute the ground truth cellwise gene regulatory networks after simulation. Also outputs ground truth bulk (entire dataset) regulatory network. Increases compute time significantly.
+    compute_rna_velocity: boolean, optional (default: False)
+           Whether or not to compute the ground truth propensity ratios after simulation. Increases compute time significantly.
+    n_jobs: int, optional (default: 8)
+           Number of cores to use.
+    random_state: int, optional (default: None)
+           Fixes seed for simulation generator.
+    lib_loc: string, optional (default: None)
+           Library directory from which to load dyngen.
+           If None, defaults to the first element of .libPaths()
+    verbose: boolean, optional (default: True)
+           Data generation verbosity.
+             
+    Returns
+    -------
+    Dictionary data of pd.DataFrames:
+    ...data['cell_info']: pd.DataFrame, shape (n_cells, 7)
```

Comment:
What does `...` mean here? I don't think this docstring will translate well as written -- `Returns` should be formatted the same way as `Parameters`